docs/releases.md

```diff
@@ -6,11 +6,81 @@ sidebar_label: Releases
 
 ------
 
+## 2.6.0 - Feb 15 2021
+
+<br><font size="4">Latest Release</font><br/> (Recommended)<br/>
+
+OpenEBS v2.6 contains some key enhancements and several fixes for the issues reported by the user community across all 9 types of OpenEBS volumes.
+
+- 3 replicated types - [Mayastor](https://mayastor.gitbook.io/introduction/), [cStor](https://github.com/openebs/cstor-operators) and [Jiva](https://docs.openebs.io/docs/next/jivaguide.html). 
+- 5 types of Local PV backed by [host path](https://docs.openebs.io/docs/next/uglocalpv-hostpath.html), [device](https://docs.openebs.io/docs/next/uglocalpv-device.html), [ZFS](https://github.com/openebs/zfs-localpv), [LVM](https://github.com/openebs/lvm-localpv) and [Rawfile](https://github.com/openebs/rawfile-localpv), and 
+- [RWX volumes using NFS](https://github.com/openebs/dynamic-nfs-provisioner). 
+
+Here are some of the key highlights in this release.
+
+### New capabilities
+
+- OpenEBS is introducing a new CSI driver for dynamic provisioning of Jiva volumes. This driver is released as alpha and currently supports the following additional features compared to the non-CSI jiva volumes. 
+   * Jiva Replicas are backed by OpenEBS host path volumes
+   * Auto-remount of volumes that are marked read-only by iSCSI client due to intermittent network issues
+   * Handle the case of multi-attach error sometimes seen on on-premise clusters
+   * A custom resource for Jiva volumes to help with easy access to the volume status
+   
+  For instructions on how to set up and use the Jiva CSI driver, please see. https://github.com/openebs/jiva-operator.
+
+### Key Improvements
+
+- Several bug fixes to the Mayastor volumes along with improvements to the API documentation. See [Mayastor release notes](https://github.com/openebs/Mayastor/releases/tag/v0.7.1). 
+- Enhanced the [NFS Dynamic Provisioner](https://github.com/openebs/dynamic-nfs-provisioner) to support using Cluster IP for dynamically provisioned NFS server. It was observed that on some of the Kubernetes clusters the kubelet or the node trying to mount the NFS volume was unable to resolve the cluster local service. 
+- [ZFS Local PV](https://github.com/openebs/zfs-localpv) added support for resizing of the raw block volumes. 
+- [LVM Local PV](https://github.com/openebs/lvm-localpv) is enhanced with additional features and some key bug fixes like:
+  * Raw block volume support
+  * Snapshot support
+  * Ability to schedule based on the capacity of the volumes provisioned
+  * Ensure that LVM volume create and deletion functions are idempotent
```

Comment:
either `create` -> `creation` or we can replace `deletion` -> `delete`. This is for the sentence beautification.